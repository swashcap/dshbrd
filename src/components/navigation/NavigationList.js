// @flow
import styled from "styled-components";

const NavigationList = styled.ul`
  background: rgba(0, 0, 0, 0.1);
  border-radius: 6px;
  display: flex;
  list-style: none;
  margin: 0;
  padding: 0 1em;
`;

export default NavigationList;
