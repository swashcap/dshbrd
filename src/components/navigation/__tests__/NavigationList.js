// @flow
import * as React from "react";
import TestRenderer from "react-test-renderer";

import NavigationList from "../NavigationList";

test("renders correctly", () => {
  const testRenderer = TestRenderer.create(
    <NavigationList>
      <li>item</li>
    </NavigationList>
  );
  expect(testRenderer).toMatchSnapshot();
});
