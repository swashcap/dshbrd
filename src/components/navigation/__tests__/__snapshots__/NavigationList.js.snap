// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  background: rgba(0,0,0,0.1);
  border-radius: 6px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  list-style: none;
  margin: 0;
  padding: 0 1em;
}

<ul
  className="c0"
>
  <li>
    item
  </li>
</ul>
`;
